marquez_client/marquez.py

```diff
@@ -36,6 +34,7 @@ def __init__(self, host=None, port=None, timeout=None):
             str(port) if port else port)
         t = os.environ.get('MARQUEZ_TIMEOUT') or str(timeout)
         self.timeout = int(t) if t.isnumeric() else self.DEFAULT_TIMEOUT_SEC
```

Comment:
Okay cool, I'll adjust it so that the timeout that the user specifies is in milliseconds. 
The value that'll have in `self.timeout` will be the conversion into a float representing seconds.